packaging/pep517_backend/_backend.py

```diff
@@ -156,10 +162,11 @@ def build_sdist(  # noqa: WPS210, WPS430
 def get_requires_for_build_sdist(
         config_settings: dict[str, str] | None = None,
 ) -> list[str]:
-    build_manpages_requested = BUILD_MANPAGES_CONFIG_SETTING in (
-        config_settings or {}
+    config_settings = config_settings or {}
+    build_manpages_requested = (
+        BUILD_MANPAGES_CONFIG_SETTING in config_settings or
+        BUILD_MANPAGES_CONFIG_ENVVAR
     )
-    build_manpages_requested = True  # FIXME: Once pypa/build#559 is addressed.
```

Comment:
If this work-around is being removed, we might as well remove the `--build-manpages` option entirely, since it's not functional. The `package-data` sanity test, as well as the `canonical-pep517-self-packaging` integration test will need to be updated to use the env var as well.

Reply:
https://github.com/ansible/ansible/pull/80363/commits/0c4d0fde4fd58eb8e3c0975f3de26c55996f4883